source/agora/network/NetworkManager.d

```diff
@@ -410,9 +409,6 @@ public class NetworkManager
 
         while (!this.peerLimitReached())
         {
-            scope (success)
-                this.taskman.wait(this.node_config.retry_delay.msecs);
```

Comment:
There's a small difference here: It won't sleep after the `continue` on L412. But I'm wondering, isn't there an infinite loop there ?

Reply:
Yes until `peerLimitReached()` is true.

Reply:
But it's a busy loop, before it wasn't

Reply:
It's the same as before?

https://github.com/bpfkorea/agora/blob/aef86ff0151c36ff83e9af9d05ec77fc7b75fd65/source/agora/network/NetworkManager.d#L208-L212